src/bootstrap/format.rs

```diff
@@ -44,6 +44,94 @@ fn rustfmt(src: &Path, rustfmt: &Path, paths: &[PathBuf], check: bool) -> impl F
     }
 }
 
+fn get_rustfmt_version(build: &Builder<'_>) -> Option<(String, PathBuf)> {
+    let stamp_file = {
+        let mut s = build.out.clone();
+        s.push("rustfmt.stamp");
+        s
+    };
```

Comment:
```suggestion
    let stamp_file = build.out.join("rustfmt.stamp");
```

Reply:
I searched the rustdoc for this function, but couldn't find it. Hopefully I will remember it exists and how it is named.
Maybe link to it from the `push` docs ?

Reply:
Can you open an issue, please?

Reply:
Done at #106219